ui/src/app/app.component.html

```diff
@@ -124,6 +125,9 @@
         <td>
           <button *ngIf="download.value.status == 'error'" type="button" class="btn btn-link" (click)="retryDownload(download.key, download.value.url, download.value.quality)"><fa-icon [icon]="faRedoAlt"></fa-icon></button>
         </td>
+        <td>
+          <a *ngIf="!!download.value.filename; else noDownloadLink" href="download/{{download.value.filename | encodeURIComponent}}" download target="_blank"><fa-icon [icon]="faDownload"></fa-icon></a>
```

Comment:
do you need the `target` property when you have a `download` link? it's not going to open the file, just download it, so the target seems irrelevant, unless I'm missing something

Reply:
Yep target is irrelevent. Only the keyword "download"